src/lib/logger/vercel/v4/vercelLoggerV4.test.ts

```
@@ -0,0 +1,1501 @@
+								analysisType: z.enum(["mean", "median", "mode", "range"]).describe("Type of analysis"),
+							}),
+							execute: async ({ data, analysisType }) => {
+								switch (analysisType) {
+									case "mean":
+										return {
+											result: data.reduce((a, b) => a + b, 0) / data.length,
+											type: "mean",
+											dataPoints: data.length,
+										};
+									case "median":
+										const sorted = [...data].sort((a, b) => a - b);
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)